src/index.js

```diff
@@ -48,6 +72,49 @@ class LumigoPlugin {
 		await this.wrapFunctions();
 	}
 
+	async getLatestNodeLayerVersionArn(layerArn) {
+		const resp = await http.get(nodeLayerVersionsUrl);
+		const pattern = `${layerArn}:(\\d*)`;
```

Comment:
Why `*` and not `+`
Any special reason for using grouping?

Reply:
good shout